tests/unittest_case.c

```diff
@@ -0,0 +1,48 @@
+#include <check.h>
+
+#include "case.h"
+
+START_TEST(test_case_diffs)
+{
+  ck_assert_uint_eq(case_diffs("abc", "abc"), 0);
+  ck_assert_uint_ne(case_diffs("aba", "ab"), 0);
+  ck_assert_uint_eq(case_diffs("abc", "AbC"), 0);
+  ck_assert_uint_eq(case_diffs("abc", "ABC"), 0);
+  ck_assert_uint_eq(case_diffs("Abc", "abC"), 0);
+  ck_assert_uint_ne(case_diffs("ä", "Ä"), 0);
+  ck_assert_uint_ne(case_diffs("", ""), 1);
```

Comment:
Another example or two, please, of strings that `case_diffs()` would say 1 about